packages/react-native-web/src/modules/createDOMProps/index.js

```diff
@@ -146,19 +112,21 @@ const createDOMProps = (component, props, styleResolver) => {
 
   // STYLE
   // Resolve React Native styles to optimized browser equivalent
-  const reactNativeStyle = [
-    component === 'a' && resetStyles.link,
-    component === 'button' && resetStyles.button,
-    role === 'heading' && resetStyles.heading,
-    component === 'ul' && resetStyles.list,
-    role === 'button' && !disabled && resetStyles.ariaButton,
+  const reactNativeStyle = StyleSheet.compose(
```

Comment:
This can now resolve to a single number for simple `<View style={styles.root} />` uses, which reduces allocations and is a slightly faster path in the style resolver (but was never hit because this code was always passing an array).